test/testlib/nuodb_database_utilities.go

```diff
@@ -19,6 +19,7 @@ type ExtractedOptions struct {
 	NrSmNoHotCopyPods int
 	NrSmPods          int
 	DbName            string
+	ClusterName		  string
```

Comment:
The new line contains a single TAB instead of SPACES. It looks OK in GoLand (on my machine), but it is screwed up in the review. Can you fix it to be the same as all the lines above?

Reply:
Sure - I actually hate the number of whitespace-only changes these different IDEs are causing.